xunit-autolabeler-v2/ast_parser/python/constants.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Google LLC.
```

Comment:
This file is not in the `source_parsers` package, but is necessary for tests to pass (since `source_parsers` draws constants from here).